vendor-local/src/legal-docs

```diff
@@ -1 +1 @@
-Subproject commit f58ed34946907d108f44f829167de904d0675324
+Subproject commit 21758916986f033b4368f6311b0acbc2bdbeb0dd
```

Comment:
you might be right.  I know I did not intentionally change any files in that directory.  I am not actually sure what to do about it as this isn't a specific file.  I am not really familiar with subprojects.